bin/openapitest.js

```diff
@@ -34,7 +34,13 @@ program
         'Will generate the html report or not.  e.g: 1 or 0',
         parseInt
     )
+      .option(
+        '-i, --inline-report-assets <n>',
+        'Inline assets in generated HTML report.',
+        parseInt
+      )
     .option('-u, --url [url]', 'Server URL. e.g: http://localhost:9000')
+    .option('-p, --proxy [proxy]', 'Proxy URL. e.g: http://127.0.0.1:8080')
```

Comment:
Adding options to the commander, this is the older version of commander, so it might look a little odd.